builtin/providers/aws/import_aws_db_event_subscription_test.go

```diff
@@ -0,0 +1,33 @@
+package aws
+
+import (
+	"os"
+	"testing"
+
+	"github.com/hashicorp/terraform/helper/resource"
+)
+
+func TestAccAWSDBEventSubscription_importBasic(t *testing.T) {
+	oldvar := os.Getenv("AWS_DEFAULT_REGION")
```

Comment:
Hi @AMeng,

This is only required if the tests have:

```
provider "aws" {
    region = "us-east-1"
}
```

Our default tests always run in us-west-2